### DISCUSSION TITLE
Trigger YT Music radio from automation

### DISCUSSION THREAD

**Discussion by patienttruth** [music-assistant/support#2058] [Q&A] [answered]
Is there a way to play a dynamically generated playlist from outside of MA? 

I.e. I can click play radio from a YT Music artist or album within MA, but don't see anything like that in the media selection area of Home Assistant.

**Comment by OzGav** [comment]
https://music-assistant.io/ui/#track-menu

**Reply by patienttruth** [reply]
Is it possible to start that from a home assistant automation instead of within the MA interface?

**Reply by OzGav** [reply, marked as answer]
Most questions are answered in the documentation https://music-assistant.io/faq/massplaymedia/

**Reply by patienttruth** [reply]
Thank you so much! I was so focused on the provider side of the docs that I skipped right through the basics. I appreciate you showing me what I should have found myself. 🙇

**Reply by kaiserkiwi** [reply]
What can I do if the service is not there? I have all Media Player Services, but not the `mass.` services.

**Reply by OzGav** [reply]
Have you installed the integration from HACS?

**Reply by kaiserkiwi** [reply]
I did not. I absolutely missed that part. The services are now there. Thank you!

**Reply by patienttruth** [reply]
I made a similar mistake after updating the add-on. I had the integration (HACS) installed, but the add-on update required an updated integration and I was getting an error. After I couple days of troubleshooting I asked and realized the detail I had missed. All is good now.

**Comment by patienttruth** [comment]
> Most questions are answered in the documentation https://music-assistant.io/faq/massplaymedia/
